### PR TITLE
Feature: sort publications by oldest updated first #1364

### PR DESCRIPTION
**Pull request by verheyenkoen**
This PR adds sorting to both the publication and dataset overview on date updated ascending as described in #1364.

Already merged the branch from PR #1388 here to take advantage of local testing. So it's best to merge #1388 before this one.

### CODE REVIEW

**Comment by nics** [comment]
@verheyenkoen #1388 is a pull against dev, this one is against main

File: static/js/app.js
Comment (verheyenkoen) [inline static/js/app.js:1]: Not sure why assets got rebuilt, didn't change anything in this PR.

**Comment by verheyenkoen** [comment]
> @verheyenkoen #1388 is a pull against dev, this one is against main

@nics Resolved.

**Comment by mietcls** [comment]
✅ Visual and functional approval (thanks for the alignment detail)

**Review by mietcls — APPROVED** [review]

**Comment by nics** [comment]
@verheyenkoen will you merge after conflicts are resolved?

**Pull request merged**